packages/css/src/components/form/form.template.ts

```diff
@@ -0,0 +1,21 @@
+import { Form } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+
+import { formGroupInputTemplate } from './form-groups/input/form-group-input.template';
+
+export function formTemplate({ mode, formGroups }: Form<TemplateResult>) {
+  return html`
+    <form class="form-${mode}">
+      <fieldset>
+        <legend>Form ${mode}</legend>
```

Comment:
`legend` moet een property op `Form` in `form.models.ts` worden.

Reply:
Nee: Ik wil formulieren kunnen maken met `formTemplate()` en dan zelf bepalen welke legend dat formulier krijgt.

Reply:
aangepast. denk ik :)